### DISCUSSION TITLE
How to internationalise gui

### DISCUSSION THREAD

**Discussion by rizh42** [apache/airflow#17825] [Q&A] [answered]
Hello
My team wants me to translate gui og airflow into our language
Where I can find all GUI's text or is there any instruments to so this?

**Comment by potiuk** [comment, marked as answer]
I don't think the current UI has any i18n support, sorry. 

But I believe the new "Modern UI" we have plans to release before Airflow 3, written in react, will have the internationalisation support. Am I correct @ryanahamilton  ?

**Reply by ryanahamilton** [reply]
i18n is definitely on the radar. Ideally we'll have language files configured in a manner that makes it fairly easy/straightforward for community members to contribute new languages.

**Reply by potiuk** [reply]
Count me in for Polish translation - it's gonna be fun to figure out some of those translations :)

**Reply by 865881900** [reply]
![image](https://user-images.githubusercontent.com/48711726/133887205-ec0e0e4d-dc1a-4cb3-a649-4627b308699e.png)
I see in flask-AppBuilder there are a lot of Po files, including Chinese, how do I start them;

**Reply by 865881900** [reply]
I still can't support it now
